qml/components/ProfileThumbnail.qml

```diff
@@ -106,7 +106,6 @@ Item {
                 height: parent.height - Theme.paddingSmall
                 anchors.centerIn: parent
                 source: profileThumbnail.photoData.local.path
-
```

Comment:
ahem, I totally planned to remove this empty line… because it's very relevant to the issue, of course.

Reply:
:D